src/System.Text.Json/tests/Serialization/Null.ReadTests.cs

```diff
@@ -29,6 +29,25 @@ public static void RootObjectIsNull()
             }
         }
 
+        [Fact]
+        public static void RootArrayIsNull()
```

Comment:
It might be good to add a test for an array within a class as well.

```C#
public class Foo
{
   public int[] Array {get; set;}
}

Foo obj = JsonSerializer.Parse<Foo>("{null}");
```

Reply:
The above test throws a `JsonException` because `"{null}"` isn't a valid JSON string, but I'd be happy to add this test if you'd like. It would probably belong here: 
https://github.com/dotnet/corefx/blob/d0d6a734fc877b76f766548d5e79a63aba5160cf/src/System.Text.Json/tests/Serialization/Object.ReadTests.cs#L152

As far as testing a valid JSON object that contains a null array, it looks like that's covered by the following, which deserializes an object with various types of properties set to null (including an array): 
https://github.com/dotnet/corefx/blob/d0d6a734fc877b76f766548d5e79a63aba5160cf/src/System.Text.Json/tests/Serialization/Null.ReadTests.cs#L33

Reply:
Ah yes, you are right. Good point! I meant testing with a json payload like `{"Array": null}`, which you rightfully pointed out, is covered.